mautic/mautic
# Beginnings of Twig Support in Custom Themes

**Ready to Merge**

I'm still writing up the documentation, but that will be added to the doc repo, not here with the code. 

For a preview of the documentation, see https://github.com/dongilbert/mautic/wiki/Converting-PHP-Templates-to-Twig

**Testing**

- Check out this PR, then clear your application cache.
- Create a new landing page with the Mauve Theme
- Use builder to ensure all areas are editable and save properly
- View landing page in frontend
- Create a new email with the Mauve Theme
- Use builder to ensure all areas are editable and save properly
- Preview email on frontend
- Test converting the coffee theme config.php to json by using the `app/console mautic:theme:json-config --theme=coffee` command from the command line.
